### DISCUSSION TITLE
How do I fix the SVG icon question marks in version 7?

### DISCUSSION THREAD

**Discussion by troycarpenter** [ampache/ampache#3982] [Q&A] [answered]
I spun up the develop docker image (not preview) and did get all the question marks in my interface, which is mentioned in the pre-release information (https://github.com/ampache/ampache/wiki/ampache7-for-admins#material-icons-have-replaced-a-lot-of-the-old-png-files).

What isn't mentioned, or I'm just having a hard time thinking this morning, is how to fix it.  What do I need to do in order to get the actual new icons?

**Comment by rsinfotech** [comment, marked as answer]
I ran into this issue during the merge of the develop branch into patch7.

I think you need to install, build and run npm to manage all the javascript with the new version.

It's described in the guide here: https://github.com/ampache/ampache/wiki/ampache7-for-admins#npm-is-required-for-js-package-installation

Well worth the upgrade, it looks great!

**Comment by troycarpenter** [comment]
I would have expected the docker container to already have that done.  Perhaps it hasn't been and won't be in the docker image until the final 7.0 is released.

**Reply by lachlan-00** [reply]
building fixed develop containers now. i added npm once but didn't update the second one where the build command copies the icon files over

**Comment by mitchray** [comment]
My understanding is its the same as the `develop` branch in the main Ampache repo, so this is to be expected

**Comment by lachlan-00** [comment]
i forgot the second command in the docker file. npm run build

that was added after the original npm packages were added and i haven't updated the docker file so i need to redo those

**Reply by lachlan-00** [reply]
Built!
